Map Rework Meta

Dunrab opened the pull request:
# Description

A lot of map fixes for Meta

- fully redid atmos to include a speerate area for the teg including a burn chamber
- gave atmos an addtional burn chamber for gas mixing
- moved the fourth locker into atmos for the fourth atmos tech
- added ID computers to each heads office
- added three shadowkin portals around the station, also added four job slots for anomalys
- added a station anchor becuase the room for it had it missing for some reason?
- replaced parameds lockers to include their hardsuits since they did not have any
- added tiny fans to arrivals airlocks
- added tiny fans to evac airlocks
- added tiny fans to escape pod airlocks since they were also missing
- added virodrobe and genedrobe to virology for doctors/medical staff that like those clothing options
- added two more perma job spawns for a total of three
- added senior roles & inventory specialist
- changed the librians door from service access to their proper door
- added proper lockers for clown, mime, bartenders and janitors
- added seb to epi
- added tropico to atmos
- added tiny fans to chefs freezer 
- added two medical borg job spawns to medical
- added a lot of missing station beacons
- removed the WT from hos's office
- raised minPlayers to 45 from 10
- moved map from /Maps/ to /Maps/Floof/
- added missing wires HV wires by the TEG SMES
- added anti-anomaly spawner in atmos gas chambers and TEG burn chamber
- added some more lights to maints
- added drawers/filing cabinets that contain paperwork templates to each front counter and head offices
- added missing at logi docking airlocks
- added missing vendomats to epi/robotics
- added three additional dorms
- added a boxer job spawn and slot
- added a pressure release valve to the TEG burn loop
- improved the bar by replacing some tables, counters and chairs with some of the newer sprites
- fixed a power issue in the hallway to evac
- fixed the power in the mantis office (the apc was not connected and no LV was ran)
- gave the mantis a fax machine 
- added directional buttons to dorms so the doors can be bolted
- added air vents to dorms since they were lacking them
- added all justice department roles
- added justice dept key to telecoms
- epi now has access to the prober and server
- made perma bedrooms more private
- tweaked perma a little bit
- expanded the salvage dock and added conveyors to move crates into the station easier
- turned the area that was called the "holodeck" into a pool
- security now has 4 sec jetpacks in the armory
- removed gamer loot from medical maints, replaced it with a tiny maints room
- added an APC for IPCs to recharge at in one of the new dorm rooms, its marked with a fancy borg sign
- added reporter jobslot and role
- gave chem a piece of plasma
- other misc changes

---

# TODO

- [x] Finish cleaning up atmos
- [x] Verify i didnt miss re-adding tiny fans to logi/salv
- [x] test the singulo/tesla setup to ensure it wont instantly loose (this may have just been people being silly and not a mapping issue)
- [x] add missing walls behind the singulo/tesla setup since they are missing
- [x] double check every department and make sure they got some paper work drawers by their front desks
- [x] a lot of maints lockers by medbay are double placed, need to remove the extra ones
- [x] dorms needs directional buttons to bolt the doors
- [x] dorms needs air vents in each room
- [x] some of the dorm rooms are very tiny so i a few more dorms should be added probably north of the boxing ring
- [x] convert part of maints near the boxing ring into a room for a boxer so we can have that job available for anyone who wants it on this station
- [x] changes requested in discord by SouthBridge
- [x] add justice department telecom server and or key to a telecom server
- [x] reporter? forget if this station has one
- [x] give chem a piece of plasma so they do not have to grind plasteel every round


---

<details><summary><h1>Map Renders</h1></summary>
<p>

![Meta-0](https://github.com/user-attachments/assets/7e2034fe-923c-4803-84b4-f23a7da7d1f7)


</p>
</details>

---

# Changelog

:cl:
- tweak: Tweaked Meta Station to fix some of the issues, see discord for a full list of changes/fixes.

Dunrab committed:
Updated meta.yml

Added senior roles
Added Anomaly roles
Moved the map path to /Maps/Floof/
Raised the minPlayers from 10 to 40

Dunrab committed:
first round of map tweaks

As of this screenshot:

fully redid atmos to include a separate area for the teg including a burn chamber
gave atmos an additional burn chamber for gas mixing
moved the fourth locker into atmos for the fourth atmos tech
added ID computers to each heads office
added three shadowkin portals around the station, also added four job slots for anomalies
added a station anchor because the room for it had it missing for some reason?
replaced parameds lockers to include their hardsuits since they did not have any
added tiny fans to arrivals airlocks
added tiny fans to evac airlocks
added tiny fans to escape pod airlocks since they were also missing
added virodrobe and genedrobe to virology for doctors/medical staff that like those clothing options
added two more perma job spawns for a total of three
added senior roles & inventory specialist
changed the librians door from service access to their proper door
added proper lockers for clown, mime, bartenders and janitors
added seb to epi
added tropico to atmos
added tiny fans to chefs freezer 
added two medical borg job spawns to medical
added a lot of missing station beacons

Superlagg commented:
Nice! Also i think the HoP's office lacks the sort of filing cabinets that have permits in them. That or I've been missing them all this time.

Dunrab commented:
> Nice! Also i think the HoP's office lacks the sort of filing cabinets that have permits in them. That or I've been missing them all this time.

I should just double check every department and make sure they got some paper work drawers by their front desks

Dunrab committed:
more tweaks

added missing wires HV wires by the TEG SMES
added anti-anomaly spawner in atmos gas chambers and TEG burn chamber
finished cleaned up of atmos
added some more lights to maints
added drawers/filing cabinets that contain paperowrk templates to each front counter and head offices
added missing at logi docking airlocks
added missing vendomats to epi/robotics

Dunrab committed:
add a slot for boxer

added a slot for a boxer since the station has a boxing ring

Dunrab committed:
additional tweaks

added three additional dorms
added a boxer job spawn and slot
finished atmos clean up
added a pressure release valve to the TEG burn loop
improved the bar by replacing some tables, counters and chairs with some of the newer sprites
fixed a power issue in the hallway to evac
fixed the power in the mantis office (the apc was not connected and no LV was ran)
gave the mantis a fax machine 
added directional buttons to dorms so the doors can be bolted
added air vents to dorms since they were lacking them

Dunrab committed:
add all Justice department roles

add all Justice department roles

Dunrab committed:
justice dept roles and other changes

justice dept roles and other changes

Dunrab committed:
justice dept telecom key

added a justice dept telecom key to telecoms since it was missing

Dunrab committed:
add a reporter job slot

add a reporter job slot

Dunrab committed:
add a report room and job slot

add a report room and job slot

Dunrab committed:
fix my silly mistake

Dunrab committed:
add a report room and job slot to meta

should be right this time

southbridge-fur commented:
You'll want to merge master to fix the test fail

Dunrab committed:
Merge branch 'Fansana:master' into map-rework-meta

Dunrab commented:
> You'll want to merge master to fix the test fail

Sure thing

Dunrab committed:
clerk needed a fax machine

Dunrab committed:
chem was missing a piece of plasma

chem on met had to take a apart a table to get plasma or run into virology so added a piece for them
